src/lando/main/auth.py

```diff
@@ -41,3 +48,84 @@ def create_user(self, claims: dict) -> User:
     def update_user(self, user: User, claims: dict) -> User:
         self.post_auth_hook(user, claims)
         return super().update_user(user, claims)
+
+
+def require_authenticated_user(f):
+    """
+    Decorator which requires a user to be authenticated.
+
+    Raises a PermissionError if a request is by an unauthenticated user.
+    """
+
+    def wrapper(request, *args, **kwargs):
+        if not request.user.is_authenticated:
+            raise PermissionError
```

Comment:
```suggestion
            raise PermissionError("Authentication is required.")
```

or something like that.